test/utils.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO(kjin): This file should supercede plugins/common.ts.
+
+import * as assert from 'assert';
+
+import {TraceSpan} from '../src/trace-span';
+
+/**
+ * Constants
+ */
+
+// The duration to give a span when it's important
+export const DEFAULT_SPAN_DURATION = 200;
+// The acceptable window of variation in span duration
+export const ASSERT_SPAN_TIME_TOLERANCE_MS = 40;
+export const SERVER_SPAN_PREDICATE = (span: TraceSpan) =>
```

Comment:
Any reason to use arrow functions for this? And it looks weird to use all-capital for a function name.

I find it easier to read to use a regular function definition for top-level functions.

Reply:
Personally I've been using arrows unless they're class members -- but it's true that it's less readable. I've changed this (and the other functions)

Reply:
Thanks. I think Google's TS style guide recommends that we prefer function declarations in this case:

> Prefer function declarations over assigning a function expression into a local variable (const x = function() {...};). TypeScript already disallows rebinding functions, so preventing overwriting a function declaration by using const is unnecessary.